### DISCUSSION TITLE
Slack bridge - is there a way to get a working bridge without "Slack App (Classic)" ?

### DISCUSSION THREAD

**Discussion by MrJohanssons** [42wim/matterbridge#1895] [Help / Q&A]
Hi 

We have a group we would like to chat with that uses Slack. We suggested that we use matterbridge to communicate between Mattermost <----> Slack. But they came back and told us that their org doesn't allow classic apps since they don't have granular permissions. 

Is there any other way to setup the Slack bot?

**Comment by realaaa** [comment]
have same requirement (cannot even create Classic app now as I understand) but for integration with Nextcloud
